src/absences/absences.test.ts

```diff
@@ -37,7 +40,58 @@ describe('Absences', () => {
 
   test('readById', async () => {
     mock.onGet(`${readPath}/1`).reply(200, { Success: true, NumResults: 1, Results: [{}] });
-    resultSingle = absences.readById(1);
-    await resultSingle.then((result) => expect(result).toStrictEqual({}));
+    const result: Absence = await absences.readById(1);
+    expect(result).toStrictEqual({});
+  });
+
+  test('create', async () => {
+    const absenceCreate: AbsenceCreate = { user_id: 1, from_date: '2020-01-01', to_date: '2020-01-03', type_id: 2, subtype_id: 3 };
```

Comment:
As said, the benefit would be detecting mistakes in the endpoint file, if we had any transformations there. For example, change the returned absence in create() and replace user_id with id. Then the test with "real data" fails and shows the diff, while the one with empty object would pass anyway.

However, I agree that it's not necessary for now as I don't foresee adding any transformations anyway. So, we could also remove this "real" data again. Let me know what you would prefer

Reply:
I would remove it because we had to maintain the "real data". Integration tests will ensure that the data types are correct. And when we have transformations we can test them with an own set of unit tests.

Reply:
Removed it